File: CS/WindowsFormsApplication3/CustomStateData.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraTreeList;
using DevExpress.XtraTreeList.Handler;
using DevExpress.XtraTreeList.Nodes;
using DevExpress.XtraTreeList.ViewInfo;
using DevExpress.Utils.Drawing;
using DevExpress.XtraTreeList.Painter;
using System.Reflection;

namespace WindowsFormsApplication3 {
    public class CustomStateData : StateData {
        public CustomStateData(TreeList treeList) : base(treeList) { }

        public Bitmap GetNodeDragBitmapEx(TreeListNode node, int visibleIndex, int indentWidth) {
            return null;
        }
    }
}

File: VB/WindowsFormsApplication3/CustomStateData.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraTreeList
Imports DevExpress.XtraTreeList.Handler
Imports DevExpress.XtraTreeList.Nodes
Imports DevExpress.XtraTreeList.ViewInfo
Imports DevExpress.Utils.Drawing
Imports DevExpress.XtraTreeList.Painter
Imports System.Reflection

Namespace WindowsFormsApplication3
	Public Class CustomStateData
		Inherits StateData
		Public Sub New(ByVal treeList As TreeList)
			MyBase.New(treeList)
		End Sub

		Public Function GetNodeDragBitmapEx(ByVal node As TreeListNode, ByVal visibleIndex As Integer, ByVal indentWidth As Integer) As Bitmap
			Return Nothing
		End Function
	End Class
End Namespace
